nixos/modules/virtualisation/qemu-vm.nix

```diff
@@ -275,6 +275,16 @@ in
           '';
       };
 
+    virtualisation.msize =
+      mkOption {
+        default = null;
+        type = types.nullOr types.ints.unsigned;
+        description =
+          ''
+            msize used for mounting 9p file systems, in bytes.
```

Comment:
maybe consider explaining what `msize` is.

Reply:
Is this better?

Reply:
Maybe also indicate what is the intended effect. (Increases performance at the expense of ????)
This is also a way to motivate the whole PR ;)

Reply:
Is this better then?

Reply:
LGTM, thanks